virt_lightning/shell.py

```diff
@@ -4,9 +4,11 @@
 import argparse
 import asyncio
 import logging
+import http.client
```

Comment:
Did you see urllib?

Reply:
I don't want any extra dependence. I can use urllib.request instead. What do you think?